src/mockdata/mockTimeEntry.ts

```diff
@@ -0,0 +1,15 @@
+import { TimeEntry } from '../types/visualisation';
```

Comment:
Will the mocked data be random everytime we run this? If it is, we will end up with the same bug as we had a couple weeks ago where the UI-tests will think that the design has been updated every time you re-render the graphs using this as a data source, which would not be fun. Is there a way to make this data predictable?

Reply:
No.

Reply:
I will implement a fix now.

Reply:
Resolved.